SPEC_MAINTAINER_PROTOCOL.md

```diff
@@ -0,0 +1,80 @@
+# Spec Maintainer Protocol
+
+> We have a protocol to maintain specs of protocols! How meta, but very useful.
+
+## Motivation
```

Comment:
There are some good ideas here but they have little to do with the spec maintainer protocol. The motivation behind the spec maintainer protocol is:

> Specs are only useful if they're maintained. Without a maintainer, a spec will stagnate and implementations of that spec will eventually move on and diverge.

Or, maybe

> Specs are only useful if there exists a protocol for iterating on them...